lib/MusicBrainz/Server/Edit/Work/RelatedEntities.pm

```diff
@@ -13,12 +13,29 @@ around '_build_related_entities' => sub
         $self->c->model('Work')->get_by_ids($self->work_ids)
     };
 
+    my ($recordings, $recording_hits) = $self->c->model('Recording')->find_by_works($self->work_ids);
+    my @recordings = @$recordings;
+
+    my @recording_ids = map { $_->id } @recordings;
```

Comment:
You can remove the `my @recordings` declaration above and just do
```suggestion
    my @recording_ids = map { $_->id } @$recordings;
```
since this is the only place it needs to be dereferenced.